km/km_cpu_init.c

```diff
@@ -385,16 +394,22 @@ km_vcpu_t* km_vcpu_fetch_by_tid(pid_t tid)
    return NULL;
 }
 
+static inline void km_signal_vcpu_stop(km_vcpu_t* vcpu)
+{
+   sigval_t val = {.sival_int = vcpu->vcpu_id | 0x10000};
```

Comment:
Maybe something symbolic instead of hardcoded 0x10000?

Reply:
I thought about it. Better yet, structure with a short and an enum. But then its needs to be in some shared .h file and so forth ... I thought the function name is symbolic enough.